index.php

```diff
@@ -23,6 +23,10 @@
     ini_set('display_startup_errors', 1);
     ini_set('display_errors', 1);
     error_reporting(-1);
+} else {
+    ini_set('display_startup_errors', 0);
+    ini_set('display_errors', 0);
+    error_reporting(null);
```

Comment:
error_reporting should still be enabled I think? We don't want errors to be displayed on the page, but we still want to log them